pkgs/web_socket/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.1.5
```

Comment:
Why the version jump? Should this be `0.1.1`?

Reply:
Yes, I hadn't read the pre-1.0 semver convention:

Although semantic versioning doesn't promise any compatibility between versions prior to 1.0.0, the Dart community convention is to treat those versions semantically as well. The interpretation of each number is just shifted down one slot: going from 0.1.2 to 0.2.0 indicates a breaking change, going to 0.1.3 indicates a new feature, and going to 0.1.2+1 indicates a change that doesn't affect the public API. For simplicity's sake, avoid using + after the version reaches 1.0.0.